### PR TITLE
WiFi on (PoE) Ethernet boards

### PR DESCRIPTION
**Pull request by df2jh**
Added two options to is-cfg.json:
- enable WiFi on (PoE) Ethernet boards as proposed in issue, closing #116
- adjust LoRa receiver gain, closing #115

### CODE REVIEW

File: data/is-cfg.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
 	"wifi": {
 		"AP": [
 			{
+		"active": false,
```
Comment (peterus) [inline data/is-cfg.json:14]: please adjust spacing

File: src/project_configuration.h [hunk 2]
```diff
@@ -21,13 +21,14 @@ class Configuration {
 
   class Wifi {
   public:
+    bool active;
```
Comment (peterus) [inline src/project_configuration.h:24]: please move the active variable to the list some lines below.

**Review by peterus — CHANGES_REQUESTED** [review]

File: data/is-cfg.json (hunk 1 above)
Comment (df2jh) [reply data/is-cfg.json]: THX, the <"active": false> line must go above <AP>, change has been commited

File: src/project_configuration.h (hunk 2 above)
Comment (df2jh) [reply src/project_configuration.h:27]: Adjusted formatting of class WiFi like class  APRS_IS

File: lib/LoRa_APRS/LoRa_APRS.h [hunk 3]
```diff
@@ -19,13 +19,16 @@ class LoRa_APRS : public LoRaClass {
...
+  void setRxGain(uint8_t gain);
+
   void setTxFrequency(long frequency);
   long getTxFrequency() const;
 
 private:
   std::shared_ptr<APRSMessage> _LastReceivedMsg;
   long                         _RxFrequency;
   long                         _TxFrequency;
+  uint8_t                      _RxGain;
```
Comment (peterus) [inline lib/LoRa_APRS/LoRa_APRS.h:31]: there is no need to have the gain in this class.
please set the gain in the `setRxGain()` function and not save it in the class. this variable is not even read somewhere.

**Review by peterus — CHANGES_REQUESTED** [review]

File: lib/LoRa_APRS/LoRa_APRS.h (hunk 3 above)
Comment (df2jh) [reply lib/LoRa_APRS/LoRa_APRS.h]: Fixed this. And set the default for gain_rx to "0", i.e. AGC, in project_configuration.cpp

Comment (peterus) [reply lib/LoRa_APRS/LoRa_APRS.h]: please fix the value also in the json file

Comment (df2jh) [reply lib/LoRa_APRS/LoRa_APRS.h]: Done

**Review by peterus — APPROVED** [review]

**Comment by peterus** [comment]
@df2jh thanks for the changes 👍

**Pull request merged**